src/test/java/com/twilio/sdk/resource/instance/taskrouter/WorkflowTest.java

```diff
@@ -95,6 +95,45 @@ public void testCreateWorkflowWithHelper() throws Exception {
 		assertNull(workflow.getFallbackAssignmentCallbackUrl());
 		assertEquals(120, workflow.getTaskReservationTimeout().intValue());
 	}
+	
+	@Test
+		public void testSerializeWorkflowFriendlyName() throws Exception {
```

Comment:
this test validates that `friendly_name` is re-serialized into `friendly_name` but we should really be testing that `filter_friendly_name` into `friendly_name` - the intended fix to the bug